### PR TITLE
Create PyConSys

### PR DESCRIPTION
**Pull request by mck-sbs**
Help wanted

### CODE REVIEW

**Review by mck-sbs — COMMENTED** [review]
file extansion fixed

**Review by mck-sbs — COMMENTED** [review]
Site fixed

File: _data/projects/PyConSys.yml [hunk 1]
```diff
@@ -0,0 +1,12 @@
...
+site: https://github.com/mck-sbs/PyConSys
+tags:
+  - python
+  - ai
+  - pid
+  - control-loop
+  - evolutional-algorithm
+upforgrabs:
+  name: help wanted
+  link: https://github.com/mck-sbs/PyConSys/projects/1
```
Comment (ritwik12) [inline _data/projects/PyConSys.yml:12]: @mck-sbs Please use this link `https://github.com/mck-sbs/PyConSys/labels/help%20wanted` for labels and also add issues in your Repo marked with this label.

File: _data/projects/PyConSys.yml [hunk 2]
```diff
@@ -0,0 +1,12 @@
+name: PyConSys
+desc: Control System with Python and Artificial Intelligence (Evolutionary Algorithm)
+site: https://github.com/mck-sbs/PyConSys
+tags:
+  - python
+  - ai
```
Comment (ritwik12) [inline _data/projects/PyConSys.yml:6]: @mck-sbs Add `artificial-intelligence` too to make it more searchable

**Review by ritwik12 — CHANGES_REQUESTED** [review]

**Comment by mck-sbs** [comment]
Sorry, deleted my local repo. I started a new pull request. Thank you.

**Pull request closed without merging**